packages/auth/src/authClients/supabase.ts

```diff
@@ -8,56 +8,72 @@ export type SupabaseUser = User
 export interface AuthClientSupabase extends AuthClient {
   /**
    * Log in an existing user, or login via a third-party provider.
+   *
    * @param options The user login details.
    * @param options.email The user's email address.
    * @param options.password The user's password.
    * @param { 'azure' | 'bitbucket' | 'facebook' | 'github' | 'gitlab' | 'google' } options.provider One of the providers supported by GoTrue.
+   * @param redirectTo A URL or mobile address to send the user to after they are confirmed.
+   * @param scopes A space-separated list of scopes granted to the OAuth application.
    */
   login(options: {
     email?: string | undefined
     password?: string | undefined
     provider?: Provider
+    redirectTo?: string
+    scopes?: string
   }): Promise<{
-    data: Session | null
+    session: Session | null
     user: User | null
     provider?: Provider
     url?: string | null
     error: Error | null
+    data: Session | null // Deprecated
   }>
   logout(): Promise<{ error: Error | null }>
   /**
    * Creates a new user.
+   *
    * @param options The user login details.
    * @param options.email The user's email address.
    * @param options.password The user's password.
+   * @param redirectTo A URL or mobile address to send the user to after they are confirmed.
    */
   signup(options: {
     email: string
     password: string
+    redirectTo?: string
   }): Promise<{
-    // data: Session | null
     user: User | null
+    session: Session | null
     error: Error | null
+    data: Session | User | null // Deprecated
   }>
+  /**
+   * Restore authentication when an OAuth or magiclink callback
+   * redirects back to site with access token and restore the session
```

Comment:
> Restore authentication ... and restore the session

Are those two different things? Or the same thing, just described using different words?

Reply:
The Redwood authentication is restored by restoring the Supabase "session". I can see how that is confusing I'll rephrase.